### PR TITLE
[WIP] Add DoubleResurrection strategy

### PR DESCRIPTION
**Pull request by souravsingh**
The PR adds DoubleResurrection strategy to Axelrod. 
I have added a WIP tag to check if the tests pass properly.

### CODE REVIEW

File: axelrod/tests/strategies/test_resurrection.py [hunk 1]
```diff
@@ -33,3 +33,29 @@ def test_strategy(self):
...
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        # Starts by Cooperating
+        self.first_play_test(C)
```
Comment (drvinceknight) [inline axelrod/tests/strategies/test_resurrection.py]: Would you be able to re write these tests using the new framework? 

I know you started writing this before the change but we'd appreciate it :)

http://axelrod.readthedocs.io/en/latest/tutorials/contributing/strategy/writing_test_for_the_new_strategy.html

File: axelrod/tests/strategies/test_resurrection.py [hunk 2]
```diff
@@ -33,3 +33,30 @@ def test_strategy(self):
         
         # Check for four defections and a cooperation by opponent
         self.responses_test([D], [C] * 5, [D, D, D, D, D, C])
+        
+class TestDoubleResurrection(TestPlayer):
+
+    name = "DoubleResurrection"
+    player = axelrod.DoubleResurrection
+    expected_classifier = {
+        'memory_depth': 1,
```
Comment (marcharper) [inline axelrod/tests/strategies/test_resurrection.py]: `memory_depth` should be 5

File: axelrod/tests/strategies/test_resurrection.py [hunk 3]
```diff
@@ -33,3 +33,30 @@ def test_strategy(self):
...
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        # Starts by Cooperating
+        self.first_play_test(C)
+        
+        actions = [(C, C), (C, D), (D, C), (C, D), (D, C)]
```
Comment (marcharper) [inline axelrod/tests/strategies/test_resurrection.py]: Can we make sure we have tests that check both 5 round depth behaviors? I think the tests will need to be at least 6 rounds of history to make sure they are triggered.

**Comment by drvinceknight** [comment]
@souravsingh I've pushed those extra tests and fixed the memory length :+1:

**Comment by marcharper** [comment]
What happened here? Looks like we need a rebase?

**Comment by drvinceknight** [comment]
> Looks like we need a rebase?

Done.

**Pull request merged**